.travis.yml

```diff
@@ -72,7 +74,7 @@ matrix:
         - cd $TRAVIS_BUILD_DIR/external/js/kyber
         - npm install
         - npm run build
-        - npm link
+        - npm run link
```

Comment:
If someone types "npm link" out of habit, will private give them a useful error?

I'm really not happy about this non-standard package format, but I don't know what you could have done to avoid it...

Reply:
Well the only other way is to import everything from the root (and this export everything up to the root). For historical reasons, the relative path when importing is the node_modules folder and you can't change that.